pom.xml

```diff
@@ -256,6 +256,87 @@
+                                        -Xms128m -Xmx1G -XX:MaxPermSize=128M
+                                        -Dhazelcast.version.check.enabled=false
+                                        -Dhazelcast.mancenter.enabled=false
+                                        -Dhazelcast.logging.type=none
+                                        -Dhazelcast.test.use.network=false
+                                        -Dhazelcast.test.multiple.jvm=true
+                                    </argLine>
+                                    <includes>
+                                        <include>**/**.java</include>
+                                    </includes>
+                                    <excludes>
+                                        <exclude>**/jsr/**.java</exclude>
+                                    </excludes>
+                                    <groups>com.hazelcast.test.annotation.ParallelTest AND com.hazelcast.test.annotation.QuickTest</groups>
+                                    <excludedGroups>
+                                        com.hazelcast.test.annotation.SlowTest,com.hazelcast.test.annotation.NightlyTest
+                                    </excludedGroups>
+                                    <systemPropertyVariables>
+                                        <multipleJVM>true</multipleJVM>
+                                    </systemPropertyVariables>
+                                </configuration>
+                            </execution>
+
+
+                            <execution>
+                                <id>singlejvm</id>
+                                <phase>test</phase>
+                                <goals>
+                                    <goal>test</goal>
+                                </goals>
+                                <configuration combine.self="override">
+                                    <redirectTestOutputToFile>true</redirectTestOutputToFile>
+
+                                    <!-- the argLine variable is needed for jacoco-->
+                                    <argLine>
+                                        -Xms128m -Xmx1G -XX:MaxPermSize=128M
+                                        -Dhazelcast.version.check.enabled=false
+                                        -Dhazelcast.mancenter.enabled=false
+                                        -Dhazelcast.logging.type=none
+                                        -Dhazelcast.test.use.network=false
+                                    </argLine>
+                                    <includes>
+                                        <include>**/**.java</include>
+                                    </includes>
+                                    <excludes>
+                                        <exclude>**/jsr/**.java</exclude>
+                                    </excludes>
+                                    <groups>com.hazelcast.test.annotation.QuickTest</groups>
```

Comment:
the singleJvm execution runs `QuickTest` only if they are not in the `ParallelTest` category. The multipleJvm runs them only if then ARE in the `ParallelTest` category.